packages/kujua-sms/views/lib/smsparser.js

```diff
@@ -1,4 +1,5 @@
 var utils = require('kujua-utils'),
+    standardiseDigits = require('eurodig'),
```

Comment:
Interesting!  How to do it without breaking the backend?  Just in-line the code?